Update $urltransform documentation (#767)
AdguardTeam/KnowledgeBase: docs/general/ad-filtering/create-own-filters.md

is its input, unchanged. Formally:

```text
urltransform = transforms
transforms = transform | transform "|" transforms
transform = substitute | decode
substitute = "/" regexp "/" replacement "/" modifiers
decode = "b64" | "pct"
```

- **`substitute`** is similar to replacement with regular expressions [in Perl](https://perldoc.perl.org/perlrequick.html#Search-and-replace).
    - **`regexp`** — a regular expression.
    - **`replacement`** — a string that replaces whatever is matched by `regexp`. `$1`, `$2`, etc. in the replacement string are replaced with the contents of the corresponding capture group.
    - **`modifiers`** — regular expression flags, e.g. `i` for case-insensitive search.
- **`b64`** — decodes a [Base64-encoded](https://datatracker.ietf.org/doc/html/rfc4648) string, both the default and the URL-safe alphabets are supported.
- **`pct`** — decodes a [percent-encoded](https://datatracker.ietf.org/doc/html/rfc3986#section-2.1) string.

In the `$urltransform` value, two characters must be escaped: the comma `,` and the dollar sign `$`. Use the backslash character `\` for this. For example, an escaped comma looks like this: `\,`.

:::info Compatibility

> AdGuard products which use a version of [CoreLibs] older than 1.20 only support a single

Reviewer: ```suggestion
AdGuard products that use a [CoreLibs] version older than 1.20 only support a single
```

> `substitute` transformation as the value of the `$urltransform` modifier.

Reviewer: ```suggestion
`substitute` transformation for the value of the `$urltransform` modifier.
```
